### DISCUSSION TITLE
Bind TinaCLI to all addresses

### DISCUSSION THREAD

**Discussion by dgaiero** [tinacms/tinacms#4769] [General]
I am running the Tina CLI inside a development container, and I cannot seem to connect to the webpage. Is there an option to specify the bind host? For astro, I can run `astro dev --host`, does the Tiny CLI have such an option?

**Comment by kldavis4** [comment]
Add `build: { host: '0.0.0.0' }` to the tina config

**Reply by ericgrossman5455** [reply]
Thanks, that did the trick! @kldavis4

**Comment by dgaiero** [comment]
Thanks for the quick response - I'm going to go ahead and close this discussion!
